CMakeLists.txt

```diff
@@ -102,9 +102,10 @@ option(DISABLE_UPENN "disable use of UPENN filetypes" OFF)
 
 if(NOT DISABLE_ITK)
    # See if we can find a compiled version of ITK (http://www.itk.org/)
-   find_package(ITK 4.9 QUIET)
+   find_package(ITK 4.9)
    if (ITK_FOUND)
       include(${ITK_USE_FILE})
+      set(HAVE_ITK ON)
```

Comment:
In CMake, we can just as well stick to `ITK_FOUND`. Let's not introduce another variable. (I guess we have `HAVE_JSON` but we don't have it for any of the others)

Reply:
I think we are using HAVE_ITK already in places. I was just not sure where it was being set and maybe added this line to be certain it was correct. See e.g. https://github.com/UCL/STIR/blob/27172be8d7d953f9167a80595404195a28a817a5/src/test/CMakeLists.txt#L117 or https://github.com/UCL/STIR/blob/27172be8d7d953f9167a80595404195a28a817a5/src/IO/IO_registries.cxx#L76

Reply:
ok, in CMake files, swith to `ITK_FOUND`, but not in cxx files (there it is set via `config.h.in`, which looks correct to me)

Reply:
perfect!